resources/lang/nl/auth.php

```diff
@@ -3,5 +3,5 @@
 return [
     'failed'   => 'Deze combinatie van e-mailadres en wachtwoord is niet geldig.',
     'password' => 'Het opgegeven wachtwoord is onjuist.',
-    'throttle' => 'Te veel mislukte loginpogingen. Probeer het over :seconds seconden nogmaals.',
+    'throttle' => 'Teveel mislukte aanmeldpogingen. Probeer het over nog eens over :seconds seconden.',
```

Comment:
Changes in this file will be overwritten by sync with [Laravel-Lang/Lang](https://github.com/Laravel-Lang/lang). Consider contributing to that repo too.

Reply:
Good idea, will do.